src/main/java/Autopark/Service/TechnicalSpecialist.java

```diff
@@ -1,11 +1,20 @@
-package Service;
+package Autopark.Service;
 
-import Vehicle.Color;
-import Vehicle.VehicleType;
+import Autopark.Infrastructure.core.annotations.InitMethod;
+import Autopark.Vehicle.Color;
+import Autopark.Vehicle.VehicleType;
 
 public class TechnicalSpecialist {
     public static final int LOWER_LIMIT_MANUFACTURE_YEAR = 1886;
 
+    public TechnicalSpecialist() {
+    }
+
+    @InitMethod
```

Comment:
Why are you using here this empty init method?

Reply:
fixed